content/en/docs/components/notebooks/container-images.md

```diff
@@ -68,8 +68,24 @@ For Kubeflow Notebooks to work with a container image, the image must:
   - the UID of `jovyan` should be `1000`
 - start successfully with an empty PVC mounted at `/home/jovyan`:
   - kubeflow mounts a PVC at `/home/jovyan` to keep state across Pod restarts
-  
+
+### Group Requirements
+Kubeflow Notebooks images are grouped by server type to ensure networking is configured correctly for Kubeflow usage. Therefore, when using a custom image that is natively supported, it should be manually added to the appropriate group during creation or as an entry to the notebook [configuration file](https://github.com/kubeflow/manifests/blob/f40fefafab30834515889d712697e915eeca123b/apps/jupyter/jupyter-web-app/upstream/base/configs/spawner_ui_config.yaml#L18).
+
+- Jupyterlab images should reside in the notebook images group "jupyterlab"  
+<img src="/docs/images/notebook-images-group-jupyterlab.png" 
+     alt="description"  
```

Comment:
Can we add alternate text for all the images for accessibility?

Reply:
Yes great idea.  
I added a small description of what the images represent for all the images.